fili-core/src/test/groovy/com/yahoo/bard/webservice/metadata/DataSourceMetadataLoadTaskSpec.groovy

```diff
@@ -170,10 +145,10 @@ class DataSourceMetadataLoadTaskSpec extends Specification {
         druidWS.jsonResponse = {fullDataSourceMetadataJson}
 
         expectedIntervalsMap = [:]
-        dimensions123.each {
-            expectedIntervalsMap.put(new DimensionColumn(dimensionDict.findByApiName(it)), [interval123] as Set)
+        dimensions.each {
```

Comment:
Rather than building an empty map, and then using an `each` to populate it, I would initialize it from the beginning:

```groovy
expectedIntervalsMap = dimensions123
              .collect { dimensionDict.findByApiName(it) }
              .collectEntries{[(it): [interval123] as Set]} 
```

And then use assignment syntax for the metrics:

```groovy
    metrics123
            .collect { new MetricColumn(it)}
            .each { expectedIntervalsMap[it] = [intervals.interval123] as Set }
```

Reply:
@archolewa Thank you very much for teaching me this good practice! I'll take a note on this :wink: